backend/server.js

```diff
@@ -284,17 +289,17 @@ function main() {
     //------------------------------------------------------------------------------
     // json poop
     //------------------------------------------------------------------------------
-    importParameters("recordings/test_parameters.json")
+    importParameters("../recordings/test_parameters.json")
```

Comment:
Did we triple check what this path is relative from? Does this break, e.g., npm start?